sdk/data/azappconfig/response_types.go

```diff
@@ -78,3 +79,70 @@ type SetSettingResponse struct {
 	// SyncToken contains the value returned in the Sync-Token header.
 	SyncToken SyncToken
 }
+type ArchiveSnapshotResponse struct {
+	generated.Snapshot
```

Comment:
I don't think we want to directly expose the generated type. I presume we'll want to define a `Snapshot` type (similar to how we did for `Setting`) so it can have the shape we want.